store/sqlstore/channel_member_history_store.go

```diff
@@ -46,15 +49,16 @@ func (s SqlChannelMemberHistoryStore) LogJoinEvent(userId string, channelId stri
 }
 
 func (s SqlChannelMemberHistoryStore) LogLeaveEvent(userId string, channelId string, leaveTime int64) error {
-	query := `
-		UPDATE ChannelMemberHistory
-		SET LeaveTime = :LeaveTime
-		WHERE UserId = :UserId
-		AND ChannelId = :ChannelId
-		AND LeaveTime IS NULL`
-
-	params := map[string]interface{}{"UserId": userId, "ChannelId": channelId, "LeaveTime": leaveTime}
-	sqlResult, err := s.GetMaster().Exec(query, params)
+	query, params, err := s.getQueryBuilder().
+		Update("ChannelMemberHistory").
+		Set("LeaveTime", leaveTime).
+		Where(sq.Eq{"UserId": userId}).
```

Comment:
Is it possible to use Sq.AND here instead of having 3 different where statements? Just to keep it constant with how it's being done later on.